docs/monitoring/tag-support.md

```diff
@@ -5,47 +5,64 @@ description: Asssign metadata to host/server/container with custom tags and crea
 
 To help you manage your metrics, hosts, and containers, and to help you
 create more useful dashboards, you can assign metadata to each
-host/server/container in the form of *tags*.
+host/server/container in the form of *tags*.
 
 Tags let you to organize your SPM hosts/servers/containers in different
 ways – for example by role, owner, or environment. Each tag consists of
 a key and a value, separated by the ':' character. 
 
 We recommend that you devise a set of tag keys that meet your needs for each host and to keep the tag set small and clean. Using a consistent and not overly broad set of tag keys makes it easier for you make the most of SPM and avoid chaos. Tags will help you create Alerts for hosts/servers/containers under certain tags or add dashboard widgets based on tags you have defined.
 
-## **AWS Tags Support**
+SPM supports 2 types of tags:
 
-SPM client has the ability to:
+1. Physical tags -  Physical Tag is an attribute of a data point one can use for filtering and grouping. They are sent with every data point. They can either automatically collected by agent or can be configured. e.g hostname, jvm name etc.
```

Comment:
They can either automatically collected => They can be either automatically collected

Reply:
e.g => e.g.,

Reply:
also command after "jvm name"